app/Http/Controllers/UserController.php

```diff
@@ -90,35 +91,99 @@ public function show()
 
```

Comment:
I dont see the logic to create user, is this feature implemented?

Reply:
for create user, its store method in UserController